node/lib/util/cherry_pick_util.js

```diff
@@ -743,10 +745,27 @@ exports.rewriteCommit = co.wrap(function *(repo, commit) {
 
     yield exports.closeSubs(opener, picks);
 
+    // Check whether conflicts exist
+    let conflictsExist = false;
+ 
+    // Doesn't Exist
+    if (!!conflicts && Object.keys(conflicts).length !== 0) {
+        conflictsExist = true;
+    }
+
     Object.keys(conflicts).sort().forEach(name => {
         errorMessage += SubmoduleRebaseUtil.subConflictErrorMessage(name);
     });
 
+    if (conflictsExist) {
+    errorMessage += `\
```

Comment:
Please indent this line by 4 spaces.

Reply:
Also, please delete any trailing whitespace.  Some of the new lines have spaces at the end.  There's probably an editor setting that will show you these, or you can find them using grep:
    grep " $" file_name